release/models/policy-forwarding/openconfig-policy-forwarding.yang

```diff
@@ -62,14 +62,20 @@ module openconfig-policy-forwarding {
 
     A forwarding-policy specifies the match criteria that it intends
     to use to determine the packets that it reroutes - this may
-    consist of a number of criteria, such as DSCP. The action of the
-    policy results in a forwarding action being applied to matching
-    packets. For example, decapsulating the packet from a GRE header.
-    In order to enact the policy based on particular interfaces - the
-    forwarding-policy is applied to an interface via referencing it
+    consist of a number of criteria, such as DSCP. The match criteria
+    is specified as rules.  If no rules are specified, then the policy
+    will match all packets.
+
+    The action of the policy results in a forwarding action being applied
+    to matching packets. For example, decapsulating the packet from a GRE
+    header. In order to enact the policy based on particular interfaces -
+    the forwarding-policy is applied to an interface via referencing it
     within an 'apply-forwarding-policy' statement associated with an
     interface.
 
+    A policy-forwarding rule with no action should be treated as an error
+    (grpc error: INVALID_ARGUMENT, netconfig error missing-element).
```

Comment:
```suggestion
    (grpc error: INVALID_ARGUMENT, NETCONF error missing-element).
```

We generally don't put in the text what the type of protocol is though.

Reply:
removed the references